test/production/unit_tests.py

```diff
@@ -0,0 +1,101 @@
+""" 
+Run me with 
+python -m unittest -v unit_tests.py
+"""
+import unittest
+import os
+import gmx2qmmm.cli
+import numpy as np
+from unittest.mock import Mock, patch
+from gmx2qmmm.generators.geometry import make_xyzq
+from gmx2qmmm.jobs.qm import QM
+# from gmx2qmmm.jobs.mm import execute_gmx, execute_gmx_communicate
+
+class TestGeometryUtils(unittest.TestCase):
+
+    def test_make_xyzq(self):
+        
+        # input data
+        coords  = [0.123, 0.456, 0.789, 0.321, 0.654, 0.987]
+        charges = [-0.57, +1.57]
+        ref_xyzq = np.array([[0.123, 0.456, 0.789, -0.57],[0.321, 0.654, 0.987, +1.57]])
+
+        # run the function
+        res_xyzq = make_xyzq(coords,charges)
+
+        # check it
+        are_equal = np.allclose(res_xyzq, ref_xyzq)
+        self.assertTrue(are_equal)
+
+
+
+
+class TestProductionLevel(unittest.TestCase):
+    # Production / Integration tests to be implemented by Alina
+    def __init__(self):
+        self.correct_output_path = 'ref_output'
+
+    @staticmethod
+    def execute_gmx2qmmm_mocking(directory = None):
+        if not directory:
+# os.chdir(os.path.join('tests', 'opt_input'))
```

Comment:
Is everything alright here? If the comments don't mean anything, please remove them